twitter-clone/app/src/main/java/io/github/vnicius/twitterclone/data/datasource/searchtweets/SearchTweetsDataSource.kt

```diff
@@ -0,0 +1,75 @@
+package io.github.vnicius.twitterclone.data.datasource.searchtweets
+
+import android.arch.lifecycle.MutableLiveData
+import android.arch.paging.PageKeyedDataSource
+import android.util.Log
+import io.github.vnicius.twitterclone.data.repository.tweet.TweetRepository
+import io.github.vnicius.twitterclone.utils.LogTagsUtils
+import io.github.vnicius.twitterclone.utils.State
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.launch
+import twitter4j.Query
+import twitter4j.Status
+import twitter4j.TwitterException
+
+class SearchTweetsDataSource(
+    val queryText: String,
+    val itemsCount: Int,
```

Comment:
If the `itemsCount` means the page size I think it makes more sense to use the name `pageSize`.